common/doc/common-doc.md

```diff
@@ -3,8 +3,15 @@
 ## Table of content
 
 * [Pool](#Pool)
+* [Grpc](#Grpc)
 
 
 ## Pool
 
 After a fork, only caller thread is activated in child process, so we mustn't join others. That's why thread container is dynamically allocated and not freed in case of fork.
+
+
+## Grpc
+The goal of the two classes provided, grpc_server_base and grpc_client_base is to create server or channel in order to use it with service. 
+grpc_server_base creates a ::grpc::server object. You can register service with third constructor parameter builder_option.
```

Comment:
For keywords like grpc_server_base, it is better to use backticks, to have something like `grpc_server_base`, otherwise, the underscores may be interpreted to write in italic, in bold...